setup.cfg

```diff
@@ -1,2 +1,48 @@
+[metadata]
+name = gene-normalizer
+version = 0.1.0
```

Comment:
can work together to figure something out -- but I think it'd be nice to define version somewhere and access it dynamically, since it's listed both here and in main.py in one of the description strings

Reply:
Ahhh you're right